src/titles.json

```diff
@@ -1121,5 +1121,33 @@
             "0004000000182200",
             "0004000000182300"
         ]
+    },
+    {
+        "name": "Mario Kart 8 Deluxe",
+        "game_server_id": "2b309e01",
+        "access_key": "09c1c475",
+        "nex_version": "4.3.2",
+        "nex_ranking_version": "4.3.2",
+        "nex_datastore_version": "4.3.2",
+        "nex_match_making_version": "4.3.2",
+        "nex_messaging_version": "4.3.2",
+        "nex_utility_version": "4.3.2",
+        "title_ids": [
+            "0100152000022000"
+        ]
+    },
+    {
+        "name": "Super Mario Maker 2",
+        "game_server_id": "22306d00",
+        "access_key": "fdf6617f",
+        "nex_version": "4.6.25",
+        "nex_ranking_version": "4.6.25",
+        "nex_datastore_version": "4.6.25",
+        "nex_match_making_version": "4.6.25",
+        "nex_messaging_version": "4.6.25",
+        "nex_utility_version": "4.6.25",
+        "title_ids": [
+            "01009B90006DC000"
+        ]
     }
```

Comment:
these fields should be added on the txt files

Reply:
I agree, I just wasn't sure how considering they have the extra field `game_server_id`

Reply:
We can probably add that to a new text file, maybe `game_server_ids.txt`? btw, maybe we could store all text files on a single folder to keep the root folder clean

Reply:
That's fair, I agree. Honestly the whole apps source is a bit of a mess. That's my fault though, I originally designed this to not even be a viewer, but rather just a decoding library. The Electron app was actually originally just an example demo. I switched gears at some point though and realized that was stupid and went all-in on the viewer concept, but that's why the source is kinda janky

Reply:
Here's a question

Should we even still maintain the .txt files?

I originally made them, and the generation script, as a way for me to quickly convert data from Kinnay's site into JSON. But seeing as how that's already been done, and we're just adding in games one at a time now, is there actually a reason to keep maintaining those extra files?

Reply:
I think the txt files could be useful if the json file gets very large to read on a text editor, but I'm fine either way.

In case we want to deprecate the TXT files, the game server id should be added for the other games btw